src/chatlog/chatlog.cpp

```diff
@@ -456,11 +456,18 @@ void ChatLog::startResizeWorker()
             workerAnchorLine = visibleLines.first();
     }
 
+    // switch to busy scene displaying the busy notification if there is a lot
+    // of text to be resized
+    int txt = 0;
+    for(ChatLine::Ptr line : lines)
```

Comment:
Interesting, that honestly makes me suspicious if the only number you were ever able to see is 0. What are your system's specs? 

I'd still feel safer quitting the loop once it hits the limit. There's no reason not to, and then at least it's bounded no matter the chat size.